### PR TITLE
Fix conda environment activate

### PR DESCRIPTION
**Pull request by JSchoenbachler**

### CODE REVIEW

**Comment by JSchoenbachler** [comment]
@jakejh I only updated the "Reproducibility" vignette, but do you want me to also update the RNA-Seq section in "Introduction" or leave that one?

**Comment by JSchoenbachler** [comment]
I also skipped the Microarray spots because we can assume people might be running that locally and not in Docker as well.

**Comment by jakejh** [comment]
I think we can leave the Intro vignette alone.

In the reproducibility vignette, though, let's be consistent between rna-seq and microarray.

File: vignettes/reproducibility.Rmd [hunk 1]
```diff
@@ -24,8 +24,9 @@ docker pull ghcr.io/hugheylab/socker
 The `seeker` package includes an example yaml file, R script, and shell script for fetching and processing a subset of an RNA-seq dataset. Copy the files to your working directory:
 
 ```{r, eval = FALSE}
+urlBase = 'https://raw.githubusercontent.com/hugheylab/seeker/master/inst/extdata/'
 for (filename in c('PRJNA600892.yml', 'run_seeker.R', 'run_seeker.sh')) {
-  file.copy(system.file('extdata', filename, package = 'seeker'), '.')}
+  download.file(paste0(urlBase, filename), file.path('.', filename))}
```
Comment (jakejh) [inline vignettes/reproducibility.Rmd]: Why is `file.path('.', ...)` necessary?

**Review by jakejh — CHANGES_REQUESTED** [review]

File: vignettes/reproducibility.Rmd (hunk 1 above)
Comment (JSchoenbachler) [reply vignettes/reproducibility.Rmd]: It's not, removed in [4f960a6](https://github.com/hugheylab/seeker/pull/30/commits/4f960a6d375774b10e0faa00de227b2af11542ce)

**Comment by JSchoenbachler** [comment]
@jakejh Changes are made!

**Review by jakejh — APPROVED** [review]

**Pull request merged**